src/header/TransferBench.hpp

```diff
@@ -608,6 +609,38 @@ namespace {
   int   constexpr MEMSET_CHAR    = 75;                 // Value to memset (char)
   float constexpr MEMSET_VAL     = 13323083.0f;        // Value to memset (double)
 
+  int GetWarpSize(std::vector<ErrResult>* errors = nullptr) {
+    hipDeviceProp_t prop;
+    hipError_t err = hipGetDeviceProperties(&prop, 0);
```

Comment:
Let's use the more focused hipDeviceGetAttribute on hipDeviceAttributeWarpSize  instead of pulling the entire device properties.